### ISSUE TITLE
SEO best practices

### ISSUE THREAD

**Issue by raae** [olavea/unlockopen.com#17]
Follow the steps in https://www.gatsbyjs.org/tutorial/part-eight/#add-page-metadata.

But use the SEO component in the layout component, not in the page template (blog-post.js). 

In the Layout component, you have to receive title and description as props. 
- `const Layout = ({ title, description, displayBio, children })`

In the page template (blog-post.js) you have add the title and description props.
- `<Layout displayBio={post.fields.bio} title={post.frontmatter.title} description={post.excerpt}>`

**Comment by olavea** [comment]
By "In the page template (blog-post.js) you have add the title and description props." You mean I have ALREADY added the title and description props?
If that is the case. Will the google Spiders from Mars understand that this is 
the bio, the title and the description? 

I assume the google Spiders from Mars will NOT understand, so I am putting it in to 
`<Layout ...`
inside blog-post.js like this:
```
    <Layout
      displayBio={post.frontmatter.bio}
      title={post.frontmatter.title}
      description={post.excerpt}
    >
```

**Comment by olavea** [comment]
about Layout.js something something

**Comment by raae** [comment]
> 
> I assume the google Spiders from Mars will NOT understand, so I am putting it in to
> `<Layout ...`
> inside blog-post.js like this:
> 
> ```
>     <Layout
>       displayBio={post.frontmatter.bio}
>       title={post.frontmatter.title}
>       description={post.excerpt}
>     >
> ```

Yes, and then in the Layout component send them to the SEO component.

**Comment by olavea** [comment]
// Why don't I make the title and desc props in SEO.js? Because I want to create every new page in blg-post.js and give them their own desc and title?

**Comment by olavea** [comment]
background_color: `#6b37bf`,
        theme_color: `#6b37bf`,
        // change the colors in manifest

**Comment by raae** [comment]
> // Why don't I make the title and desc props in SEO.js? Because I want to create every new page in blg-post.js and give them their own desc and title?

Yes
